### ISSUE TITLE
parallel I/O and python 3

### ISSUE THREAD

**Issue by mdenolle** [SeismicData/pyasdf#55]
Hello,

Is there any improvement of compatibility of the parallel I/O mode for pyasdf with python 3?

**Comment by krischer** [comment]
Yes - this has been working fine for a while. We just forgot to update the documentation. The important thing is just to install an `h5py` version with parallel I/O support (either self-compile or use a conda version, e.g. https://anaconda.org/clawpack/h5py-parallel - but there are others out there as well).

Just a note of caution: Parallel I/O is actually rarely worth it for waveform data (took me a while to learn this) and even for non waveform data it is really hard to get it to perform. HDF5 only issues truly collective read/write calls under certain conditions which are not always true for structurally complex data. Additionally the file system has to be actually parallel, the lustre stripe count (on lustre systems) has to be correct, and more complexities. Also one gives up some hdf5 niceties with parallel I/O (e.g. one cannot write compressed data in parallel). Modern SSDs can read/write at multiple GBs per second and thus I/O rarely ends up the bottleneck.

I'll leave this open as a reminder to update the documentation and also put the above note of caution there.
